compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -2844,6 +2870,24 @@ define_print_and_forward_display! {
         p!(print_def_path(self.0.def_id, self.0.args));
     }
 
+    TraitRefPrintSugared<'tcx> {
+        if !with_no_queries()
+            && let Some(kind) = cx.tcx().fn_trait_kind_from_def_id(self.0.def_id)
+            && let ty::Tuple(args) = self.0.args.type_at(1).kind()
+        {
+            p!(write("{}", kind.as_str()), "(");
```

Comment:
This interacts as expected with the `trim`/`dont_trim` macros, right? (Don't see why they wouldn't)

Reply:
It should, since those are global flags